registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/java/io/registration/processor/workflow/manager/stage/WorkflowActionJob.java

```diff
@@ -0,0 +1,266 @@
+					description.getCode() + " -- " + registrationId, e.getMessage(), e.toString());
+		} catch (Exception ex) {
+			isTransactionSuccessful = false;
+			description.setMessage(PlatformErrorMessages.RPR_WORKFLOW_ACTION_JOB_FAILED.getMessage());
+			description.setCode(PlatformErrorMessages.RPR_WORKFLOW_ACTION_JOB_FAILED.getCode());
+			regProcLogger.error(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(),
+					description.getCode() + " -- " + registrationId,
+					PlatformErrorMessages.RPR_WORKFLOW_ACTION_JOB_FAILED.getMessage() + ex.getMessage()
+							+ ExceptionUtils.getStackTrace(ex));
+			object.setInternalError(Boolean.TRUE);
+
+		} finally {
+			regProcLogger.info(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.REGISTRATIONID.toString(),
+					registrationId, description.getMessage());
+			if (isTransactionSuccessful)
+				description.setMessage(PlatformSuccessMessages.RPR_WORKFLOW_ACTION_JOB_SUCCESS.getMessage());
+
+			String eventId = isTransactionSuccessful ? EventId.RPR_402.toString() : EventId.RPR_405.toString();
+			String eventName = isTransactionSuccessful ? EventName.UPDATE.toString() : EventName.EXCEPTION.toString();
+			String eventType = isTransactionSuccessful ? EventType.BUSINESS.toString() : EventType.SYSTEM.toString();
+
+			/** Module-Id can be Both Success/Error code */
+			String moduleId = isTransactionSuccessful
+					? PlatformSuccessMessages.RPR_WORKFLOW_ACTION_JOB_SUCCESS.getCode()
+					: description.getCode();
+			String moduleName = ModuleName.WORKFLOW_ACTION_JOB.toString();
+			auditLogRequestBuilder.createAuditRequestBuilder(description.getMessage(), eventId, eventName, eventType,
+					moduleId, moduleName, registrationId);
+		}
+
+		return object;
+	}
+
+	public void processActionablePausedPackets() throws WorkflowActionException {
+		List<InternalRegistrationStatusDto> actionablePausedPacketsList = registrationStatusService.getActionablePausedPackets(fetchSize);
+		Map<String, List<InternalRegistrationStatusDto>> defaultResumeActionPacketIdsMap = new HashMap<>();
+		for (InternalRegistrationStatusDto dto : actionablePausedPacketsList) {
+			if (defaultResumeActionPacketIdsMap.containsKey(dto.getDefaultResumeAction())) {
+				List<InternalRegistrationStatusDto> internalRegistrationStatusDtos = defaultResumeActionPacketIdsMap
+						.get(dto.getDefaultResumeAction());
+				internalRegistrationStatusDtos.add(dto);
+				defaultResumeActionPacketIdsMap.put(dto.getDefaultResumeAction(), internalRegistrationStatusDtos);
+			} else {
+				List<InternalRegistrationStatusDto> internalRegistrationStatusDtos = new ArrayList<InternalRegistrationStatusDto>();
+				internalRegistrationStatusDtos.add(dto);
+				defaultResumeActionPacketIdsMap.put(dto.getDefaultResumeAction(), internalRegistrationStatusDtos);
+			}
+		}
+		for (Entry<String, List<InternalRegistrationStatusDto>> entry : defaultResumeActionPacketIdsMap.entrySet()) {
+

```

Comment:
Remove these kind of unwanted line breaks

Reply:
Fixed